uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-details-tab.component.js

```diff
@@ -219,8 +296,8 @@ define([
       }
 
       // Make a copy of the list
-      listToReturn = vm.requestDayTypes.slice(0);
-      date = vm._convertDateToServerFormat(date);
+      listToReturn = _.cloneDeep(vm.requestDayTypes);
```

Comment:
why do we need this? I don't see any modification being done on the list

Reply:
Right. I checked and it is not being mutated.